app/controllers/staffs_controller.rb

```diff
@@ -0,0 +1,39 @@
+class StaffsController < ApplicationController
+
+  before_action :authenticate_customer!
+  
+  def index
+    @staffs = Staff.all(_account_id: current_customer.current_account_id).fetch
```

Comment:
I see it's possible to get collections https://github.com/scventures/sagn-customers-web/blob/master/app/controllers/service_requests_controller.rb#L9